internal/hcs/vmcompute.go

```diff
@@ -0,0 +1,289 @@
+package hcs
```

Comment:
I think we should rename this to something like `hcscalls.go`, seems clearer as to what's in it.

Reply:
We can discuss this but thats a larger change. Ideally we would have an internal/vmcompute package that encapsulates the syscall layer of `vmcompute.dll` I think.